shell/common/snapshot_controller_impeller.cc

```diff
@@ -73,8 +73,11 @@ sk_sp<DlImage> DoMakeRasterSnapshot(
     );
   }
 
-  impeller::TextFrameDispatcher collector(context->GetContentContext(),
-                                          impeller::Matrix());
+  impeller::TextFrameDispatcher collector(
+      context->GetContentContext(),                 //
+      impeller::Matrix(),                           //
+      impeller::Rect::MakeSize(render_target_size)  //
+  );
```

Comment:
This pattern of "first dispatch to a TFD and then to a DlDispatcher" happens in a number of places. It feels like a convenience method might be nice.

Reply:
In https://github.com/flutter/engine/pull/55019 I refactored this dispatching into a reusable static method.